packages/pharos-site/src/components/statics/design-token/TokenTable.tsx

```diff
@@ -1,14 +1,11 @@
 import { tokenTable } from './TokenTable.module.css';
 import { useEffect, useState } from 'react';
 
-export const TokenTable = (
-  title: string,
-  content: JSX.Element,
-  subtitle?: JSX.Element
-): JSX.Element => {
-  const [StateTable, setStateTable] = useState<JSX.Element>(<></>);
+export const TokenTable = (props: any) => {
```

Comment:
Could you define an interface for these props and use the `FC` type for this component? That should help clear the lint warnings. You could look at some of the other site components as an example.

Reply:
Hmm. I think my IDE is lagging or something. It takes a while for the lint warnings to show.